src/pykeen/hpo/__init__.py

```diff
@@ -28,21 +28,40 @@
 ...    model='TransE',
 ... )
 
-Every model in PyKEEN not only has default hyper-parameters, but default
-strategies for optimizing these hyper-parameters. While the default values can
-be found in the ``__init__()`` function of each model, the ranges/scales can be
+Every model in PyKEEN has default values for its hyper-parameters chosen from
+the best reported values in each model's respective paper, unless otherwise
```

Comment:
Make clearer that this is the **original** paper and not any other paper which maybe tuned these values further, e.g. Rufinelli et al.

Also we could add a word of caution that these values are for a specific dataset, and might be non-optimal for other datasets.

Reply:
is that true? later we discuss how we could have also used later paper's params. How do we want to communicate this?